packages/hooks/src/useP2PPaymentMethods.ts

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { useFetch } from '@deriv/api';
+import { useStore } from '@deriv/stores';
+
+const type_to_icon_mapper = {
+    bank: 'IcCashierBankTransfer',
+    other: 'IcCashierOther',
+    ewallet: 'IcCashierEwallet',
+};
```

Comment:
perhaps we can separate these out into its own constants file?

Reply:
@ameerul-deriv but what would be the benefit of that? 🤔

This constant is only used in this file and should only be used here, Putting it in a different file will create a dependency between files and make the code harder to read as devs need to jump between files. And eventually, someone will reuse the constant for a different purpose and then make changes in it over time and here we go buys 😁

Reply:
I dont think this'll lead to more bugs in the future IMO. I don't think P2P will be changing these 3 particular payment method types as all the payment methods are sorted under these three categories. furthermore by putting the type_to_icon_mapper object into a constants file, we can reuse it across multiple files and components. since its already repeated in another file, we can follow similar practices we have done in other constants file such as buy-sell.js or floating-rate.js. And perhaps we might need to use these constants as well in other components

Reply:
Hmm the problem is this constant is inside hooks package. We dont have any additional file for constants. I dont know if its a good idea to have it there.

Reply:
ahhhh i see, yeah if this was inside p2p, it would be a different case 👍 thanks for clarifyin!

Reply:
@ameerul-deriv @farhan-nurzi-deriv Let's discuss this in person if you guys are interested 🙇🏻

Reply:
ahhh no worries i was looking at this as if this hook was inside p2p, but because its outside of p2p, your argument makes sense 😛 so you can ignore and resolve this comment @farhan-nurzi-deriv